### DISCUSSION TITLE
Custom delimiters?

### DISCUSSION THREAD

**Discussion by skylerknight** [markdoc/markdoc#60] [Q&A] [answered]
Would be great to be able to customize the delimiters from `{% %}` to something different such as `{{ }}` or `<% %>`

**Comment by mfix-stripe** [comment, marked as answer]
@skylerknight we originally had the ability to do so in Markdoc, but we removed it since that feature ran counter to one of our main goals, which it to strive for Markdoc to become a standard, idiomatic markup syntax with stronger opinions about various markdown features (e.g. only one way to create headings). Allowing users to customize the syntax (e.g. by choosing new delimiters) makes it more difficult for users to understand what "idiomatic Markdoc" even means. 🙂

**Reply by skylerknight** [reply]
Sounds fair to me. 🙂
